src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -934,8 +911,13 @@ static public ServiceType getMetaData() {
     // know they are different than the inmoov bots.
     // each bot should have their own name, it's confusing that the inmoov bots
     // are named en-US and so are the program ab bots.
-    meta.addDependency("program-ab", "program-ab-data", "1.1", "zip");
-    meta.addDependency("program-ab", "program-ab-kw", "0.0.8.4");
+
+    // meta.addDependency("program-ab", "program-ab-data", "1.2", "zip");
+    // meta.addDependency("program-ab", "program-ab-kw", "0.0.8.5");
+
+    meta.addDependency("program-ab", "program-ab-data", null, "zip");

```

Comment:
you're including program-ab-data here as a dependency, however it was removed from the pom.xml ...

Reply:
correct,
its included here with null - which will make ivy query for the latest during runtime..
its not included inside the maven, because I believe eclipse has a bug in dealing with it, while maven seems ok with parens notation (as saintandreas deps have always had)